README.adoc

```diff
@@ -148,83 +155,84 @@ Write a simple `Consumer` class that consumes the artist JSON using both JSON-P
 include::finish/src/main/java/io/openliberty/guides/consumingrest/Consumer.java[tags=!comment;!class]
 ----
 
-=== Processing JSON using JSON-P
-
-Write the method for consuming the REST service using JSON-P in the `Consumer` class:
-
-[source, java, indent=0]
-----
-include::finish/src/main/java/io/openliberty/guides/consumingrest/Consumer.java[tags=consumeWithJsonp]
-----
+=== Processing JSON using JSON-B
 
-This method makes a `GET` request to the running artist service and retrieves the JSON.
+JSON-B is a Java API that is used to serialize Java objects to JSON messages and vice versa.
 
-Write the `collectArtists` and `collectAlbums` helpers. These methods will parse that JSON and
-collect its objects into individual POJOs:
+To include the JSON-B provider in your project, add the following dependency to your `pom.xml`,
+which has been done for you:
 
-[source, java, indent=0]
+[source, xml, role="no_copy"]
 ----
-include::finish/src/main/java/io/openliberty/guides/consumingrest/Consumer.java[tags=collectArtists;collectAlbums]
+<dependency>
+   <groupId>javax.json.bind</groupId>
+   <artifactId>javax.json.bind-api</artifactId>
+   <version>1.0</version>
+</dependency>
 ----
 
-Write the `getTotalArtists` method in the `ArtistResource` class. This method will use the JSON-P
-approach to return the total number of artists present in the JSON.
-
-`start/src/main/java/io/openliberty/guides/consumingrest/service/ArtistResource.java`:
+Write the method for consuming the REST service using JSON-B in the `Consumer` class:
```

Comment:
What about, "Use JSON-B in the `Consumer` class to write the method for consuming the REST service:"?